src/netstandard/FabActUtil/Microsoft.ServiceFabric.Actors.targets

```diff
@@ -15,13 +15,17 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
     <Copy SourceFiles="$(TargetDir)\$(AssemblyName).runtimeconfig.dev.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.dev.json" SkipUnchangedFiles="true" />
     <Copy SourceFiles="$(TargetDir)\$(AssemblyName).deps.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.deps.json" SkipUnchangedFiles="true" />
+    <Copy SourceFiles="$(TargetDir)\$(AssemblyName).runtimeconfig.json" DestinationFiles="$(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.json" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(ActorServiceAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
     <Copy SourceFiles="@(FabActUtilAssemblies)" DestinationFolder="$(FabActUtilWorkingDir)" SkipUnchangedFiles="true" />
   </Target>
-  <Target Name="_UpdateServiceFabricServiceManifest" AfterTargets="Build" DependsOnTargets="_CopyAssembliesToFabActUtilWorkingDir" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
+  <Target Name="_UpdateRuntimeConfig" AfterTargets="Build" DependsOnTargets="_CopyAssembliesToFabActUtilWorkingDir" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
+    <Exec Command="powershell $(FabActUtilWorkingDir)\FixRuntimeConfig.ps1 -RuntimeConfigFile $(FabActUtilWorkingDir)\FabActUtil.runtimeconfig.json" />
+  </Target>  
+  <Target Name="_UpdateServiceFabricServiceManifest" AfterTargets="Build" DependsOnTargets="_UpdateRuntimeConfig" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
     <Exec Command="dotnet &quot;$(FabActUtilWorkingDir)\FabActUtil.dll&quot; /spp:&quot;$(ServicePackagePath)&quot; /t:manifest /sp:&quot;$(ServicePackagePrefix)&quot; /in:&quot;$(OutDir)\$(AssemblyName).dll&quot; /arp:&quot;$(TargetDir)\&quot; $(FabActUtilAdditionalArguments)" WorkingDirectory="$(MSBuildProjectDirectory)" />
   </Target>
-  <Target Name="_UpdateServiceFabricApplicationManifest" DependsOnTargets="_CopyAssembliesToFabActUtilWorkingDir" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
+  <Target Name="_UpdateServiceFabricApplicationManifest" DependsOnTargets="_UpdateRuntimeConfig" Condition=" '$(UpdateServiceFabricManifestEnabled)' == 'true' ">
```

Comment:
>_UpdateRuntimeConfig [](start = 75, length = 20)

This dependency change is not needed.

Reply:
reverted it.

Reply:
Aman, that dependency is required in both the fabactutil calls because parallel execution happens, so the runtime config isnt updated in 1 case and that fails.